import_specifications/templates/README.md

```diff
@@ -0,0 +1,74 @@
+# Instructions for import specification templates
+
+This file contains instructions for how to use the import specification templates provided
+along with the file. The templates allow a user to import many data objects to KBase without
+having to type the information into the UI by hand and may be generated by automated processes.
+
+## Prerequisites
+
+It is expected that the user is familar with the KBase Narrative interface, the staging area,
+has previously imported the data type in question via the UI, and is familiar with the import
+parameters.
+
+## Templates
+
+The templates come in three flavors: CSV (comma separated values), TSV (tab separated values),
+and Excel (`.xls` and `.xlsx` files are supported). Most users will want to select one of the
+three formats, but they can be mixed and matched as desired (but see `Troubleshooting` below).
+
+To use a template, examine the human readable fields in the template - the third row in CSV
+and TSV files, and the first visible row in Excel files (the first two rows are hidden). Each
+column corresponds to an input in the UI for the data type. Users fill in the fields in the
+template just as they would the fields in the import UI. For instance, the genbank_genome
+template contains a `GenBank File Path` column, as does the `Genome` import UI in the bulk
+import cell in the Narrative. The user would fill in rows 4 and above (2 and above in Excel)
+in the templates with the same data they would use to fill in the UI. For checkboxes, specify
+`1` for checked and `0` for unchecked.
+
+There is one caveat, in that the templates currently support specifying all the parameters for
+an import for each import, while the UI does not. Currently the first row of data in the template
+will be used to specify parameters for all the other rows for the case where the UI
+does not allow individual parameter settings per row (see note 1). Only input files and output
+object names are allowed to differ for a set of imports for a type.
+
+Once the template or templates are filled out, upload the templates and the corresponding files
+to be imported to KBase to the narrative staging area. For each template, choose the
+`Import Specification` type and ensure the selection checkbox for each file is checked. There is
+no need to choose types for or select the files that are specified in the templates - indeed
+selecting those files will cause them to be, in effect, selected twice.
+
+When the template's types have been properly chosen and the selection checkboxes are correct,
+click the `Import Selected` button. The bulk import cell will be created in your narrative and
+populated as per the contents of your selected templates.
+
+## Lookahead inputs
+
+Some importers, such as the Genbank importer, have input fields that are based on a dynamic
+lookup from user input - for example, the taxon ID of a genome is looked up from the prefix
```

Comment:
was taxon_id removed from the template set up? Or can it still be used to find the scientific name?

Reply:
`taxon_id` was removed from the user inputs, and `scientific_name` is now used to look up the taxon id in the RE. Then that value is sent to the app in the `taxon_id` field.

Reply:
Got it. I had saw it removed in #138, so was curious how that worked.